docs/source/api/apollo-server.mdx

```diff
@@ -599,6 +599,69 @@ The `start` method triggers the following actions:
 1. If your server is a [federated gateway](https://www.apollographql.com/docs/federation/managed-federation/overview/), it attempts to fetch its schema. If the fetch fails, `start` throws an error.
 2. Your server calls all of the [`serverWillStart` handlers](../integrations/plugins-event-reference/#serverwillstart) of your installed plugins. If any of these handlers throw an error, `start` throws an error.
 
+
+## `executeOperation`
+
+The async `executeOperation` method is used primarily for [testing GraphQL operations](../testing/testing/#testing-using-executeoperation) through the server's request pipeline without going through a full HTTP operation.
```

Comment:
"the server's" -> "Apollo Server's"

"without going through a full HTTP operation" -> "_without_ sending an HTTP request"

Reply:
We're missing an example call for `start` and `stop`, but we should probably include a quick one here like we have for the constructor. Maybe even define a short fake GraphQL operation in the codeblock that's being executed?